pkg/apis/app/v2/kieapp_types.go

```diff
@@ -773,7 +773,8 @@ type ServerTemplate struct {
 	// When set to true will configure the KIE Server image to disable all capabilities that are not related to decisions, Default to true for RHDM environments and false to RHPAM.
 	DecisionsOnly *bool `json:"decisionsOnly,omitempty"`
 	// KieExecutorMDBMaxSession number of KIE Executor sessions
-	KieExecutorMDBMaxSession *int `json:"kieExecutorMDBMaxSession,omitempty"`
+	KieExecutorMDBMaxSession *int   `json:"kieExecutorMDBMaxSession,omitempty"`
+	SecretAdminCredentials   string `json:"secretAdminCredentials,omitempty"`
```

Comment:
shouldn't this be only on the common config?

Reply:
Here is needed because the loop of the []servertemplate can't access to it as a part of the commonconfig

Reply:
No sure if it is true.
previous user and admin fields was readable, why this secret isn't?